psi4/src/psi4/ccresponse/optrot.cc

```diff
@@ -70,6 +70,8 @@
 #include <cstdio>
 #include <cstring>
 #include <cstdlib>
+#include <cmath> /* needed for lround*/
```

Comment:
I think these comments should go. If someone is doing a header cleanup and they remove these two the file won't compile, so I don't think they are really necessary IMO.

Reply:
Removed the comments, thanks!